rb/lib/selenium/webdriver/remote/w3c/bridge.rb

```diff
@@ -116,6 +116,20 @@ def page_source
                               'return source;')
           end
 
+          #
+          # Create a new top-level browsing context
+          # https://w3c.github.io/webdriver/#new-window
+          # @param type [String] Supports two values: 'tab' and 'window'.
+          #  Use 'tab' if you'd like the new window to share an OS-level window
+          #  with the current browsing context.
+          #  Use 'window' otherwise
+          # @return [Hash] Containing 'handle' with the value of the window handle
+          #  and 'type' with the value of the created window type
```

Comment:
Can this ever return a "window" of the type other than requested?  If not what's the point in returning the type?

Reply:
I don't believe so. This is the returned result per the webdriver spec:
https://w3c.github.io/webdriver/#new-window
```
8. Let result be a new JSON Object initialised with:

"handle"
The value of handle.
"type"
The value of type.
```